packages/react-native-sdk/docusaurus/docs/reactnative/04-ui-components/participants/local-participant-view.mdx

```diff
@@ -17,10 +24,57 @@ By default, the `LocalParticipantView` renders in a floating mode.
 
 In order to use the `LocalParticipantView` as a standalone component, you should use the following code:
 
-```tsx
+```tsx {4}
 import { LocalParticipantView } from '@stream-io/video-react-native-sdk';
 
 const App = () => {
   return <LocalParticipantView />;
 };
 ```
+
+## Props
+
+### `alignment`
+
+Determines where the floating participant video will be placed.
+
+- 0 for top-left.
+- 1 for top-right.
+- 2 for bottom-left.
+- 3 for bottom-right.
+
+| Type                | Default value  |
+| ------------------- | -------------- |
+| `0` \|`1`\|`2`\|`3` | `top-right(0)` |
```

Comment:
may be lets change this to an enum instead of const enum, kinda ugly to use in SDK for numbers,, sorry

Reply:
or just 4 types of strings is also enough

Reply:
Done @santhoshvai